src/control/xojfile/LoadHandler.cpp

```diff
@@ -532,10 +544,13 @@ void LoadHandler::parseStroke()
 	 * we've read just before. 
 	 * Afterwards, clean the read timestamp data.
 	 */
```

Comment:
This is for backwards compatibility with the .xoj files still containing audio timestamps